code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,343 @@
+            logging.error(
+                "The file %s" % f + " is not recognized as input file for mvs "
+                "check /mvs_eland/inputs/elements/default_csv for correct "
+                "file names."
+            )
+
+
+    with open(os.path.join(input_directory, output_filename), "w") as outfile:
+        json.dump(input_json, outfile, skipkeys=True, sort_keys=True, indent=4)
+    logging.info("Json file created successully from csv's and stored into"
+                 "/mvs_eland/inputs/%s" % output_filename + "\n"
+                "Scenario includes %s" % len(list_assets) + " assets of "
+                "type: %s" % list_assets)
+    logging.debug("Json created successfully from csv.")
+    if pass_back:
+        return input_json
+
+
+def create_json_from_csv(input_directory, filename, parameters):
+
+    """
+    One csv file is loaded and it's parameters are checked. The csv file is
+    then converted to a dictionary; the name of the csv file is used as the
+    main key of the dictionary. Exceptions are made for the files
+    ["economic_data", "project", "project_data", "simulation_settings"], here
+    no main key is added. Another exception is made for the file
+    "energyStorage". When this file is processed, the according "storage_"
+    files (names of the "storage_"-columns in "energyStorage" are called and
+    added to the energyStorage Dictionary.
+
+
+    :param input_directory: str
+        path of the directory where the input csv files can be found
+    :param filename: str
+        name of the inputfile that is transformed into a json, without
+        extension
+    :param parameters: list
+        List of parameters names that are required
+    :return: dict
+        the converted dictionary
+    """
+
+    logging.debug(
+        "Loading input data from csv: %s", filename
+    )
+    csv_default_directory=os.path.join(input_directory, "default_csv/")
+    df = pd.read_csv(
+        os.path.join(input_directory, "csv/", "%s.csv" % filename),
+        sep=",",
+        header=0,
+        index_col=0,
+    )
+
+    # check parameters
+    extra = list(set(parameters) ^ set(df.index))
+    if len(extra) > 0:
+        for i in extra:
+            if i in parameters:
+                logging.error(
+                    "In the file %s.csv" % filename
+                    + " the parameter "
+                    + str(i)
+                    + " is missing. "
+                    "check %s", csv_default_directory + "for correct "
+                    "parameter names."
+                )
+            else:
+                logging.error(
+                    "In the file %s.csv" % filename
+                    + " the parameter "
+                    + str(i)
+                    + " is not recognized. \n"
+                    "check %s", csv_default_directory + "for correct "
+                    "parameter names."
+                )
+
+    # convert csv to json
+    single_dict2 = {}
+    single_dict = {}
```

Comment:
Do display string of assets read from csv: 
```suggestion
    single_dict = {}
    asset_name_string = ""
```